test/stress/stress_test.cc

```diff
@@ -0,0 +1,58 @@
+#include "stress_test.h"
+
+namespace Envoy {
+namespace Stress {
+
+const std::string StressTest::ORIGIN_CLUSTER_NAME{"origin_cluster"};
+
+// TODO dump shows codec type is still HTTP1
```

Comment:
So the clients are generating HTTP1?

Reply:
Not sure.  Proposal:  next I'll add additional parameters to the test case so we can cross with http vs h2 and IPv4 vs. IPv6.   I'll make sure the right thing is being sent on the wire.

When I'm done I'll update this PR if it's still open, else I'll open a new one

Reply:
I turned this into a parameterized test that uses HTTP1 vs. HTTP2 X IPv4 vs. IPv6.    In the process I addressed this TODO